mtp_prisoner_location_admin/settings/base.py

```diff
@@ -64,6 +87,13 @@
     },
 ]
 
+
+TEMPLATE_CONTEXT_PROCESSORS = global_settings.TEMPLATE_CONTEXT_PROCESSORS + (
+    'moj_utils.context_processors.debug',
+    'django.core.context_processors.request'
+)
```

Comment:
Context processors should now be included in the ['OPTIONS']['context_processors'] attribute of the TEMPLATES setting.

The django.templates.context_processors.request entry that is already there replaces django.core.context_processors.request so you can just delete this line and add moj_utils.context_processors.debug to the list above (@line 80).